TACC/Core-Styles
# enhance: standardize message icon support

## Overview / Changes

- Add message icons as defined by past designs.
- Use message icons via mixins.

## Related

- Version 3

## Testing

1. Compare all message instances to before.
2. Verify expected changes:
    - all
        - darker msg. border / icon bkgd.
    - [admonition](http://localhost:3000/components/preview/admonition)
        - new default danger icon
    - [message--type](http://localhost:3000/components/detail/c-message--type)
        - inline message text not colored
        - spacing is smaller
        - inline message has icon (except type "info")
    - [message--type--portal](https://cep.tacc.utexas.edu/static/ui/components/detail/c-message--type-portal.html)
        - section message has icon (except type "info")
        - spacing is smaller
    - [message--type--cms](https://cep.tacc.utexas.edu/static/ui/components/detail/c-message--type-cms.html)
        - text is darker
    - [form--cms](http://localhost:3000/components/detail/c-form--cms) [form--portal](http://localhost:3000/components/detail/c-form--portal) [django-cms-forms](https://cep.tacc.utexas.edu/static/ui/components/detail/django-cms-forms.html) (click [Toggle Field Errors])
        - no change (no icon added)
    - [x-message](http://localhost:3000/components/detail/x-message--type)
        - all supported messages are listed
        - styles do not have CMS nor Portal tweaks
        - type "info" has icon

## UI

Skipped.
